ui/app/templates/vault/cluster/replication-dr-promote/details.hbs

```diff
@@ -3,22 +3,28 @@
     {{!-- ARG TODO setup error state and return --}}
     <ReplicationPage @model={{model}} as |Page|>
       <Page.header 
-        @showTabs={{false}} 
-        @model={{model}}
+        @showTabs={{false}}
       />
       <Page.toggle />
       <Page.dashboard 
       {{!-- passing in component to render so that the yielded components are flexible based on the dashboard --}}
         @componentToRender='replication-secondary-card' as |Dashboard|>
         <Dashboard.card
           @title="States"
-          @data={{model}}
         />
         <Dashboard.card
           @title="Write-Ahead Logs (WALs)"
-          @data={{model}}
         />
-        <Dashboard.rows @data={{model.dr}}/>
+        {{#if Dashboard.isSyncing }}
+          <AlertBanner 
+            @title="Syncing in progress"
+            @type="info"
+            @secondIconType="loading"
```

Comment:
i'm having trouble getting storybook running locally right now, but have you tried seeing what this looks like when `secondIconType` is set to one of the other types, such as warning or info?

Reply:
I have.  And it works as expected.  Here's secondIconType as the warning. 
![image](https://user-images.githubusercontent.com/6618863/79355022-646cd800-7efa-11ea-833f-4348729179b9.png)